components/TableCell.tsx

```diff
@@ -0,0 +1,34 @@
+import { TableCell, tableCellClasses } from '@mui/material'
```

Comment:
`components/Table` could be reused

Reply:
as https://github.com/Magickbase/godwoken-explorer-ui/blob/update-ui/components/TxList/index.tsx#L164-L249